src/style.css

```diff
@@ -0,0 +1,111 @@
+@font-face {
+  font-family: Avenir;
+  src: url(/fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(/fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+body {
+  margin: 0;
+}
+
+/* #region BIG FORM */
+
+.form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.icon {
+  position: absolute;
+  pointer-events: none;
+}
+
+.input {
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+  color: #3d4e61;
+  padding: 0;
+  width: 100%;
+
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.icon--big {
+  height: 19px;
+  width: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.input__big {
+  font-size: 16px;
+  font-weight: 900;
+  padding-left: 62px;
+  height: 70px;
+}
+
+.input__big::placeholder {
+  color: inherit;
+  font-size: 16px;
+  font-weight: 400;
+}
+
+.input__big:focus {
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  background-image: linear-gradient(#fff, #f6f6f7);
+  outline: none;
+}
```

Comment:
If input__big only a modifier of input and you have input__small modifier, so maybe you can find the same props in both and move it to block `:focus` ?

Reply:
I'll try my best)